proposals/description/clawback.ts

```diff
@@ -0,0 +1,57 @@
+import { TemplatedProposalDescription } from '@custom-types/types';
+
+const clawback: TemplatedProposalDescription = {
+  title: 'FIP-113: End Departed Rari Founders Vesting of TRIBE',
+  commands: [
+    /////////  Mint FEI and TRIBE onto new RARI timelocks
+    {
+      target: 'fei',
+      values: '0',
+      method: 'mint(address,uint256)',
+      arguments: (addresses) => [addresses.newRariInfraFeiTimelock, '3254306506849315068493151'],
```

Comment:
todo: update with the new balance after rari infra team makes a last claim

Reply:
done